src/sp/protocol/pubsub0/pub.c

```diff
@@ -73,7 +73,11 @@ pub0_sock_init(void *arg, nni_sock *ns)
 	nni_pollable_init(&sock->sendable);
 	nni_mtx_init(&sock->mtx);
 	NNI_LIST_INIT(&sock->pipes, pub0_pipe, node);
```

Comment:
This is already a *runtime* tunable via NNG_OPT_SENDBUF....
I don't think I agree with making this a compile time setting -- if you want to change it in your application just call 
`nng_sock_set_int(sock, NNG_OPT_SENDBUF, size);`